### ISSUE TITLE
More output functions for PromptPlus.Error

### ISSUE THREAD

**Issue by ividyon** [FRACerqueira/PromptPlus#66]
I noticed that PromptPlus.Error seems to only support WriteLine in terms of output. However PromptPlus has other interesting methods, such as Write, DoubleDash, SingleDash etc. I believe those should also be supported for the `outerr` stream.

**Comment by FRACerqueira** [comment]
@ividyon , I don't understand this problem well, could you make it clearer by giving an example of what you want?

**Comment by ividyon** [comment]
There are:

```cs
PromptPlus.WriteLine("text")
PromptPlus.SingleDash("text")
PromptPlus.DoubleDash("text")
// ...
```

for the regular PromptPlus console output.

This console output is directed to the `stdout` stream.

There is also

```cs
PromptPlus.Error.WriteLine("text")
```
to direct lines to the `stderr` stream, which is [good practice for diagnostic output from a console application](https://clig.dev/#the-basics).

But there are no:

```cs
PromptPlus.Error.SingleDash("text")
PromptPlus.Error.DoubleDash("text")
// ...
```
and other such printing functions.

**Comment by FRACerqueira** [comment]
Perfect, I understand!

**Comment by FRACerqueira** [comment]
@ividyon , 'm working on this implementation.

The implementation so far looks like this:

```csharp
            //write to standard error output stream for any output included within 'using'
            using (PromptPlus.OutputError())
            {
                PromptPlus.DoubleDash("anytext");
                PromptPlus.WriteLine("Test");             
            }
```

**Comment by ividyon** [comment]
That's a very good way of solving it!

**Comment by FRACerqueira** [comment]
Fixed scheduled for publication 4.0.6
